aztec/src/main/kotlin/org/wordpress/aztec/AztecTagHandler.kt

```diff
@@ -144,6 +154,12 @@ class AztecTagHandler(val context: Context, val plugins: List<IAztecPlugin> = Ar
                 handleElement(output, opening, preformatSpan)
                 return true
             }
+            INPUT -> {
+                if (opening && attributes.getValue("type") == "checkbox") {
+                    return handleCheckboxInput(output, context, attributes)
```

Comment:
In case there is a `checkbox` element in the `<ul><li>`, we change the type of the `<ul>` parent to `AztecTaskList`

Reply:
thanks for this suggestion! As discussed in DM, I've implemented it in 5b264edb51b252d3ed4b5f48aacc45a96a60add0